packages/cli/plugin-ssg/src/index.ts

```diff
@@ -47,7 +47,7 @@ export const ssgPlugin = (): CliPlugin<AppTools> => ({
         const routes = readJSONSpec(buildDir);
 
         // filter all routes not web
-        const pageRoutes = routes.filter(route => !route.isApi);
+        const pageRoutes = routes.filter(route => !route.isApi || route.isSPA);
```

Comment:
in the end of this function, there is a line `writeJSONSpec(buildDir, pageRoutes.concat(apiRoutes));`

which may cause duplicate routes...and `createServer` also use `apiRoutes`

Reply:
It can work normally, but it's not elegant haha

Reply:
Yeah, it's not elegant here, because if we want it to be relatively elegant, we either need to change the information in route.json or need to make a lot of changes to this file.

Do you have any ideas?

Reply:
If I create a new object to merge the information of pageRoutes and apiRoutes as the data source of the page to be rendered and route.json, do you think this is appropriate?

Reply:
can we juest use `route.isSPA` to judge ？